### ISSUE TITLE
add ServerASMResponses to all endpoints

### ISSUE THREAD

**Issue by kiniry** [FreeAndFair/ColoradoRLA#241]
This needs to happen tonight @dmzimmerman.

The model implementation for such is in `AuthenticateStateAdministrator`:
```
    // Build the ASM server response.
    final ServerASMResponse asm_response =
        new ServerASMResponse(Main.dosDashboardASM().currentState(),
                              Main.dosDashboardASM().enabledUIEvents());
    the_response.body(Main.GSON.toJson(asm_response));
```
We need to nest such a response inside of all other responses so that @ranweiler can parse out the enabled UI events to update the UI.

**Comment by dmzimmerman** [comment]
What do you mean by "nest"? That's not even possible with the endpoints that do streaming, as far as I can see.

**Comment by dmzimmerman** [comment]
Also... it's a bit of a chicken and egg problem, because the "current state" you want to return for the ASM is not really the state at the end of the request, until we transition the ASM at the end of the request. But by then you've already sent the response body.

**Comment by dmzimmerman** [comment]
It seems to me that the right solution might be to actually have a dedicated endpoint to just get the ASM state for a requested ASM; it's guaranteed to be right when you get it, though of course subsequent concurrent connections can change it... but that's the case anyway. And it makes things less complicated for the endpoints.

**Comment by dmzimmerman** [comment]
My alternative solution was implemented in #245
